00-default/videoeditors.rules

```diff
@@ -0,0 +1,4 @@
+# Davinci Resolve - https://www.blackmagicdesign.com/products/davinciresolve/
+# Sometimes, for some reason the process name is `GUI Thread` (verified via `btop` and `pkill` works)
+{ "name": "resolve", "type": "Heavy_CPU" }
+{ "name": "GUI Thread", "type": "Heavy_CPU" }
```

Comment:
I don't think the GUI thread needs to be added here

Reply:
![image](https://github.com/user-attachments/assets/6f22d050-20e7-419d-8bec-ca0a638267dc)
So `resolve` here would be enough?

Reply:
Yes.

Reply:
oh so we can also put in their binary name huh, ok.

Reply:
Adding "GUI Thread" is correct but is that really the process that does the rendering? The name caught me offguard so I assumed it didn't.

Reply:
yes, but sometimes it shows as normal `resolve` tho.
![image](https://github.com/user-attachments/assets/f96e27fa-1385-4a89-be2c-660f60e7259b)

Let me do a quick screencast

Reply:
https://github.com/user-attachments/assets/e83b8c24-f127-4efd-bf52-529fd8cad710

After i took the video, I double checked again and realized that
the process is indeed `resolve`, it shows as `GUI Thread` (for some reason?) because I ran it using `prime-run`. But it indeed render it using their `resovle` process

Reply:
Great! If there's nothing else I'll go ahead and merge this.

Reply:
yeah, go ahead